theme/src/resources/templates/category_pages/liferay_dxp_7_0.hbs

```diff
@@ -31,26 +31,38 @@
 					name: '{{dc '7_1-top_resources_tab'}}',
 					configs: [
 						{
-							description: '{{dc '5_2-compatibility_matrix_card_description'}}',
+							description: '{{dc '7_0-latest_release_card_description'}}',
```

Comment:
Are all the uses of the double single quotes going to parse okay? I remember you asking about a way to handle this before.

Reply:
Yeah, it's been a little of a headache. When there is a dynamic content that contains a contraction, if we use single quotes to surround the `{{dc}}` helper, it breaks. We could update everything to use double quotes but if we have dynamic content that contains double quotes like `Lorem ipsum "dolor" sit amet`, it will probably break too. Zendesk does have a `json_stringify` helper but it doesn't take another helper as an argument. So as a work around, if the dynamic content contains a contraction, I wrap it in ` `` ` for now. 

Any thoughts on alternative solutions?

Reply:
I think the template literal is a good solution, because you can still conform to the zendesk format with their single quotes.

Reply:
Is the dynamic content just a key that get's translated?  If so, don't we have the ability to update the key to whatever we want?  So if it has a contraction or a single quote in the key, we can just remove it from the key itself, but allow the translation to have the the single quote?

Like @jwu910 mentioned, using the template literal is another solution that should work for our needs.

Reply:
The dynamic content's value is actually parsed on the template and outputted when it's being passed to the javascript. So changing the key wouldn't solve the issue. 

I currently use the template literal notation for keys whose value contain contractions and have showed Cole how to check for it. We're okay for now, but always looking for more robust solutions.